crates/net/network/src/message.rs

```diff
@@ -37,6 +38,53 @@ impl NewBlockMessage {
     }
 }
 
+/// Internal form of a `PooledTransactions` message
+/// Same as [`NewPooledTransactionHashes68`] but types and sizes are Option.
+#[derive(Debug, Clone)]
+pub struct NewPooledTransactionHashes {
```

Comment:
please remove this type and instead move to separate PR

Reply:
follow it up with f53e2a7f9a19eb19ffe05dc86a74b84867a3a8c2.